src/main/java/Main.java

```diff
@@ -126,6 +113,13 @@ private Runnable notifyChannel(Store store, Integer slots) {
     }
 
     private void run() {
+
+        System.out.println(String.format("Finding stores within %s miles of zip %s", SEARCH_RADIUS_MILES, ZIP));
+
+        Map<String, Instant> stores = locateStores(ZIP).stream()
+                .collect(Collectors.toMap(Store::getId, i -> Instant.EPOCH));
+        storeIds.putAll(stores);
```

Comment:
The gist of the change. Retrieve list of stores at startup so we don't have to hard code them.